lib/modem_fota/fota_client_mgmt.c

```diff
@@ -68,25 +69,27 @@ static void http_response_cb(struct http_response *rsp,
 			     void *user_data);
 static int tls_setup(int fd, const char * const tls_hostname);
 static int do_connect(int * const fd, const char * const hostname,
-		      const uint16_t port_num, bool use_fota_apn);
+		      const uint16_t port_num, bool use_fota_apn,
+		      const char * const ip_address);
 static int parse_pending_job_response(const char * const resp_buff,
 				      struct fota_client_mgmt_job * const job);
 
-#define API_HOSTNAME "static.api.beta.nrfcloud.com"
+#define API_STATIC_IP_1 "75.2.37.83"
+#define API_STATIC_IP_2 "99.83.231.82"
```

Comment:
Should these two IP address be set by CONFIG? Recommended to use host name **#if !defined CONFIG_MODEM_FOTA_CARRIER_LIB**  #else set the host by IP address that is defined with a CONFIG item

Reply:
I think we could have a normal Kconfig item for static IP usage. I would like to have as little carrier lib specific stuff in the library as possible.

Do we need to make the IP addresses configurable? The idea is, after all, that they're static.

I will merge the changes now, we can change the configuration separately if needed.

Reply:
The idea is to not expose the IP addresses on GitHub open source

Reply:
the IP addresses are not secret, anyone can execute `nslookup static.api.nrfcloud.com` and see them.